src/DynamoCoreWpf/ViewModels/Watch3D/DefaultWatch3DViewModel.cs

```diff
@@ -610,7 +611,8 @@ protected virtual void OnRenderPackagesUpdated(NodeModel node, RenderPackageCach
             // If there is no attached model update for all render packages
             if (watchModel == null)
             {
-                if(node.IsVisible == false)
+                //When Watch3D nodes are in canvis we need to ignore this even under certain situations. 
```

Comment:
spelling (canvas) - also comment is not super clear to me?
` ignore this even under certain situations`
whats this,
what situations?

Reply:
@mjkkirschner Does this make more sence now?